hub/api/dataset.py

```diff
@@ -198,6 +213,12 @@ def pytorch(self, transform: Optional[Callable] = None, workers: int = 1):
         """
         return dataset_to_pytorch(self, transform, workers=workers)
 
+    def get_total_meta(self):
+        return {
```

Comment:
docstring? also this should probably be _get_total_meta as it's a helper function.